cli/dcoscli/package/main.py

```diff
@@ -180,6 +189,31 @@ def _add_repo(repo_name, repo_url, index):
     return 0
 
 
+def _import_repos(repos_file):
+    """Add package repos from a JSON file
+
+    :param repos_file: path to JSON file containing repos. A YAML file can
+        be used if PyYAML is installed.
+    :type repos_file: str
+    :rtype: int
+    """
+
+    package_manager = get_package_manager()
+
+    with open(repos_file) as f:
+        if yaml:
+            data = yaml.load(f)
+        else:
+            data = json.load(f)
+
+        for index, repo in enumerate(data['repositories']):
```

Comment:
Hmmm. I guess my thought was square brackets were okay here, because if the user gives a JSON file that doesn't have a `repositories` key, then we might as well raise an error. Though maybe we should have a nice error instead of a traceback.

How about 01081a2?